boot/zephyr/include/usb_dfu/usb_dfu.h

```diff
@@ -0,0 +1,7 @@
+/*
```

Comment:
this files prototypes a board interface, not really features related strictly to usb_dfu - change name.

Reply:
I have no idea how should I name...
Do you have any good idea?